### ISSUE TITLE
Not compatible with gnome 3.4

### ISSUE THREAD

**Issue by bend** [simon04/gnome-shell-extension-weather#88]
Please add support to gnome 3.4.

**Comment by simon04** [comment]
See branch `gnome3.4`.

**Comment by bend** [comment]
Yes just installed it from this branch. However the configuration pane could not be openned and the forecast keeps loading. Do you have any idea how to fix this ?

**Comment by simon04** [comment]
How did you install it? Make sure to properly install `weather-extension-configurator.py` under `/usr/bin/weather-extension-configurator` (I think this is missing in the Makefile).

**Comment by bend** [comment]
Ok that fixed the configuration panel. But forecast keeps loading. Any idea ?

**Comment by simon04** [comment]
Have you set a WOEID?

**Comment by bend** [comment]
Yep. Disabling translate weather conditions foxed the problem. However sunrise and sunset times are still not set.

**Comment by simon04** [comment]
> However sunrise and sunset times are still not set.

What do you mean by "not set"? You enabled them in the configurator, but they are not shown? (If so, please give me your WOEID)

**Comment by bend** [comment]
Sorry disabling the show sunrise/sunset option wixed the forecast problem.

**Comment by bend** [comment]
If the sunrise/sunset option is enabled, Only today weather shows and the sunrise sunset times are not shown. If I disable the sunrise/sunset option, the forecast weather shows. My WOEID is BEXX0005

**Comment by cartmanez** [comment]
I use gnome 3.4 and I have the same problem with sunset/sunrise. My woeid is BOXX0005. I tried many others and there is always the same:
If 'Show sunset/sunrise' options is disabled, extension works fine. But if it's enabled, weather is not shown and keeps loading.
There is an error in my .xsession-errors

```
(gnome-shell:1013): Json-CRITICAL **: json_object_get_string_member: assertion `node != NULL' failed
      JS LOG: A ReferenceError has occured: sunrise is not defined
```

I've checked the resulting json in browser - sunrise is there, but for some reason it's not present in the astronomy js object (method refreshWeather: function(recurse) {...}). 
Seems like it's strange parsing error.
I hope this information will help to identify the root of the problem
